src/views/CohortPage.js

```diff
@@ -39,20 +38,23 @@ const CohortPage = () => {
       </Route>
       <Route path={`${path}/minority-group`}>
         <BarGraph
-          title={`Cohort ${cohortID}: Minority Groups`}
```

Comment:
Oh, I just noticed that the Cohort ID is being removed from the graph title due to this change.

Reply:
I think it might have happened while I was merging master.